omc3/harpy/constants.py

```diff
@@ -45,3 +45,7 @@
 COL_COEFFS: str = "COEFFS"
 COL_FREQS: str = "FREQS"
 COL_TIME: str = "TIME"
+
+# Header Keys -----------------------------------------------------------------
+HEADER_AMPLITUDE_UNIT: str = "AMPLITUDE_UNIT"
+AMPLITUDE_UNIT_VALUE: str = "m"  # meter (default unit for amplitudes) Please confirm this unit is correct!
```

Comment:
I am fairly certain this is correct, but this is actually one of the few occasions where I would not make a constant. Everything is meters (after the inputs are converted) in our analysis.